<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>queryWindow</class>
 <widget class="QMainWindow" name="queryWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SQL Query</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="queryEditor" name="editQuery">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="labelQuery">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Enter Query:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="updateButton">
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QTableView" name="tableViewQuery"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Query"/>
    <addaction name="actionSave_Query"/>
    <addaction name="actionSave_Query_As"/>
    <addaction name="actionExport_to_CSV"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave_Query">
   <property name="text">
    <string>Save Query</string>
   </property>
  </action>
  <action name="actionSave_Query_As">
   <property name="text">
    <string>Save Query As...</string>
   </property>
  </action>
  <action name="actionExport_to_CSV">
   <property name="text">
    <string>Export to CSV</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOpen_Query">
   <property name="text">
    <string>Open Query</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>queryEditor</class>
   <extends>QPlainTextEdit</extends>
   <header>queryeditor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
